--- a/libdocument/ev-annotation.c
+++ b/libdocument/ev-annotation.c
@@ -518,6 +518,52 @@
 }
 
 /**
+ * ev_annotation_get_rgba:
+ * @annot: an #EvAnnotation
+ * @rgba: (out): a #GdkRGBA to be filled with the annotation color
+ *
+ * Gets the color of @annot.
+ *
+ * Since: 1.2.1
+ */
+void
+ev_annotation_get_rgba (EvAnnotation *annot,
+                        GdkRGBA      *rgba)
+{
+    g_return_if_fail (EV_IS_ANNOTATION (annot));
+    g_return_if_fail (rgba != NULL);
+
+    *rgba = annot->rgba;
+}
+
+/**
+ * ev_annotation_set_rgba:
+ * @annot: an #Evannotation
+ * @rgba: a #GdkRGBA
+ *
+ * Set the color of the annotation to @rgba.
+ *
+ * Returns: %TRUE if the color has been changed, %FALSE otherwise
+ *
+ * Since: 1.2.1
+ */
+gboolean
+ev_annotation_set_rgba (EvAnnotation  *annot,
+                        const GdkRGBA *rgba)
+{
+    g_return_val_if_fail (EV_IS_ANNOTATION (annot), FALSE);
+    g_return_val_if_fail (rgba != NULL, FALSE);
+
+    if (gdk_rgba_equal (rgba, &annot->rgba))
+        return FALSE;
+
+    annot->rgba = *rgba;
+    g_object_notify (G_OBJECT (annot), "rgba");
+
+    return TRUE;
+}
+
+/**
  * ev_annotation_get_color:
  * @annot: an #EvAnnotation
  * @color: (out): a #GdkColor to be filled with the Annotation color.
@@ -579,52 +625,6 @@
 
     return TRUE;
 }
-
-/**
- * ev_annotation_get_rgba:
- * @annot: an #EvAnnotation
- * @rgba: (out): a #GdkRGBA to be filled with the annotation color
- *
- * Gets the color of @annot.
- *
- * Since: 1.2.1
- */
-void
-ev_annotation_get_rgba (EvAnnotation *annot,
-                        GdkRGBA      *rgba)
-{
-    g_return_if_fail (EV_IS_ANNOTATION (annot));
-    g_return_if_fail (rgba != NULL);
-
-    *rgba = annot->rgba;
-}
-
-/**
- * ev_annotation_set_rgba:
- * @annot: an #Evannotation
- * @rgba: a #GdkRGBA
- *
- * Set the color of the annotation to @rgba.
- *
- * Returns: %TRUE if the color has been changed, %FALSE otherwise
- *
- * Since: 1.2.1
- */
-gboolean
-ev_annotation_set_rgba (EvAnnotation  *annot,
-                        const GdkRGBA *rgba)
-{
-    g_return_val_if_fail (EV_IS_ANNOTATION (annot), FALSE);
-    g_return_val_if_fail (rgba != NULL, FALSE);
-
-    if (gdk_rgba_equal (rgba, &annot->rgba))
-        return FALSE;
-
-    annot->rgba = *rgba;
-    g_object_notify (G_OBJECT (annot), "rgba");
-
-    return TRUE;
-}
 
 /* EvAnnotationMarkup */
 typedef struct {
